kodilla-patterns/src/main/java/com/kodilla/patterns/builder/bigmac/Bigmac.java

```diff
@@ -0,0 +1,65 @@
+package com.kodilla.patterns.builder.bigmac;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Bigmac {
+
+    private final String bun;
+    private final int burgers;
```

Comment:
czym jest int burgers? To jest typ burgera, identyfikator, czy ilosc burgerow? Zmienne powinny byc "samoopisujace sie".